doc/source/examples/pdp_regression_bike.ipynb

```diff
@@ -0,0 +1,645 @@
+{
```

Comment:
<p>Prerequisites</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi/pull/721/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>Still to fix?</p>

Reply:
<p>should be fixed already</p>

Reply:
I think there's a missing `e`

Reply:
yeah ... I fixed it now ...